source/includes/_simulator-reference.html.md

```diff
@@ -147,6 +156,39 @@ Python <simulator_file>.py --predict=1
 
 After your BRAIN is finished training it can be used to "predict" or perform in the simulation. How well it does depends on how long you let it train! Using your BRAIN involves running your simulator file as you did when training, but now in prediction mode with `--predict`. The default is `--predict=latest` which will use the version of the latest training session that you just ran. You can use a different version of your BRAIN if you have trained it multiple times by replacing `latest` with the version number.
 
+## Recording Data to File
+
+> Record file example
+
+```python
+# Excerpt of find_the_center_sim.py
+
+if __name__ == "__main__":
+    config = bonsai_ai.Config()
+    
+    config->set_record_enabled(true);
+    config->set_record_file("find_the_center.json");
+
+    brain = bonsai_ai.Brain(config)
+    sim = BasicSimulator(brain, "find_the_center_sim")
+    sim.enable_keys(["delta_t", "goal_count"], "ftc")
+
+    print('starting...')
+    last = clock() * 10000000
```

Comment:
should be `1000000` (probably a typo carried over during copy pasta)

Reply:
I'll open a diff to fix this in the codebase.